spew/bypasssafe.go

```diff
@@ -15,7 +15,7 @@
 // NOTE: Due to the following build constraints, this file will only be compiled
 // when either the code is running on Google App Engine or "-tags disableunsafe"
```

Comment:
This note is no longer accurate with the change in this PR.

Perhaps something like:

> when either the code is running on Google App Engine, on js architecture, or "-tags disableunsafe" is added to the go build command line.

Same for the other 2 files in this PR.